### DISCUSSION TITLE
Request execution limit

### DISCUSSION THREAD

**Discussion by Hyroshima** [mercadopago/sdk-js#87] [General]
Hello, I would like to know if there is any GET request execution limit per minute?

**Comment by lucmantovani** [comment]
Hello @Hyroshima!

Yes, there is a limit.
Are you receiving any 429 responses?

**Reply by Hyroshima** [reply]
I wanted to know what the limit would be because the only one I found was the request expiration time but not the quantity.
I'm creating a model that searches at specific times, so to avoid problems with the amount of request I would go per a low average from the limit

**Reply by lucmantovani** [reply]
By your use case, I imagine the [Notifications](https://www.mercadopago.com/developers/en/docs/checkout-api/additional-content/notifications/introduction) feature would be a good option.
I recommend you to have a look 🙂

Should you have any issues, you can contact our [support](https://www.mercadopago.com/developers/en/support/center/tickets) directly.

**Reply by Hyroshima** [reply]
for this model the url_notification is not useful because I'm going to use static qrcode, so the idea is to check the sending of payments at the time the person is going to pay, so within a time interval I do a check
